### ISSUE TITLE
Support of Softhyphens

### ISSUE THREAD

**Issue by PhenX** [dompdf/dompdf#104]
_Original author: mamek...@gmail.com (November 29, 2009 13:05:32)_

<b>What would you like dompdf to do:</b>
Now Softhypens (&amp;shy;) will show every time and there are no line breaks 
after a soft hyphen character.

<b>Do you have an example:</b>

``` html
<div style="width:20px">Do&shy;nau&shy;dampf&shy;schiff&shy;fahrts&shy;gesell&shy;schaft</div>
```

_Original issue: http://code.google.com/p/dompdf/issues/detail?id=97_

**Comment by 1manfactory** [comment]
No hyphenation support? Too bad, that makes dompdf useless for me.

**Comment by naitsirch** [comment]
This is fixed in #2009
